doc/building.rst

```diff
@@ -128,24 +128,30 @@ Then, in Terminal:
 
    .. code-block:: bash
 
-      % brew install git qt qtkeychain cmake openssl pkgconfig glib doxygen cmocka
+      % brew install git qt qtkeychain cmake openssl glib cmocka
```

Comment:
@FlexW with the updated CMakeLists.txt, would there be any warnings if the documentation-specific dependencies are not present when building the client?

Reply:
With the updated cmake scripts you get after the configure phase a summary that tells you which required and optional features are enabled. But anyway if the configure phase fails you get at the end an error. I think it should be clear from the output if there are any errors.

Reply:
I can follow this up if necessary once the new CMake scripts get merged.